recipes/pcaone/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.1.5" %}
+{% set system = "Linux"%}  # [x86_64 and linux]
+{% set system = "Mac"%}  # [x86_64 and osx]
+{% set sha256 = "bd06d62ef83438588f086c8e8c9e8fc44960a3560153d32a2a65ce0240343659" %} # [x86_64 and linux]
+{% set sha256 = "ccbddd6d4be5b8bd1a80bcd4cb4e29538eb168ee2a018cd44dc7d444e03ff50a" %} # [x86_64 and osx]
+
+package:
+  name: pcaone
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Zilong-Li/PCAone/releases/download/v{{ version }}/PCAone-v{{ version }}_x64-{{ system }}-iomp5.zip
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  build:
+
```

Comment:
Using precompiled binaries is a recipe for things breaking. That's fine if everything is statically compiled, but if there's any dynamic linkage (likely given the mkl references) then this will very likely break when deployed.